website/src/content/docs/docs/extending-typespec/basics.md

```diff
@@ -42,9 +42,9 @@ You can skip this step if you've used one of the templates above.
 Run the following commands:
 
 ```bash
-> mkdir myLibrary
-> cd myLibrary
-> npm init
```

Comment:
Probably need to remove the `>` characters? I don't think the way it was formatted is what we actually want here.

Reply:
Ah yeah